### PR TITLE
Add August meeting notes

### PR DESCRIPTION
**Pull request by trallard**
Adding notes as volunteered on Discord 

----
📚 Documentation preview 📚: https://docs-community--127.org.readthedocs.build/en/127/monthly-meeting/2024-08.html

### CODE REVIEW

File: docs/monthly-meeting/2024-08.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Documentation Community Team Meeting (August 2024)
+
+- **Date:** 2024-08-06
+- **Time:** [19:00 UTC](https://arewemeetingyet.com/UTC/2024-08-06/19:00/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- [**Discourse thread**](TODO) (for August)
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:6]:
```suggestion
- **This HackMD:** [https://hackmd.io/@encukou/pydocswg1](https://hackmd.io/@encukou/pydocswg1)
- [**Discourse thread**](https://discuss.python.org/t/documentation-community-meeting-tuesday-6th-august-2024/59599) (for August)
```

File: docs/monthly-meeting/2024-08.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- Ned Batchelder / `@nedbat`
+- Petr Viktorin / `@encukou`
+- Usman
+- Carol / `@willingc`
+- Daniele / `@EvilDMP`
+
+## Introductions
+
+> If there are any new people, we should do a round of introductions.
+
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:34]: We can remove this placeholder:
```suggestion
```

File: docs/monthly-meeting/2024-08.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+## Reports and celebrations
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- [Name] ...
+
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:42]: And this one:
```suggestion
```

File: docs/monthly-meeting/2024-08.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  - We've been working to not losing what we already have but also not get rid of RTD features.
+  - 3.13 or 3.14 is the version we'll migrate first, to get a feel for the workflow. All other versions will be served by python.org for now. Then we'll migrate one version at a time.
+  - [Trey] Will this enhance the search? [Manuel] There have been many improvements in RTD search lately. Traditionally RTD overwrites Sphinx search entirely with server-side (Elastic Search) search. Now there's a way to choose RTD or Sphinx search from the docs theme. There's also a person who said they're working on the Sphinx sphinx search engine.
+  - [Trey] So there won't be a change on day one.
+  - [Carol] For the PR previews, is there an option to link to the most changed file in the PR? [Manuel] We're working on it, don't know the details. The idea is to perform a diff and determine what changed, and link to it directly.
+    - Issue: https://github.com/readthedocs/readthedocs.org/issues/11319
+    - Design doc: https://github.com/readthedocs/readthedocs.org/pull/11507
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:55]:
```suggestion
    - Issue: [readthedocs/readthedocs.org#11319](https://github.com/readthedocs/readthedocs.org/issues/11319)
    - Design doc:[readthedocs/readthedocs.org#11507](https://github.com/readthedocs/readthedocs.org/pull/11507)
```

File: docs/monthly-meeting/2024-08.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  - [Carol] That's way further ahead than I thought.Thank you!
+
+## Discussion
+
+[Ezio] Deprecations in the What's New file - replacing the list of deprecations with a table
+- [Hugo] We previously talked about putting deprecations in include files. We have that now: https://docs.python.org/3/deprecations/index.html
+- Compare https://pillow.readthedocs.io/en/stable/deprecations.html and https://docs.pytest.org/en/stable/deprecations.html
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:62]:
```suggestion
- [Hugo] We previously talked about putting deprecations in include files. We have that now: [https://docs.python.org/3/deprecations/index.html](https://docs.python.org/3/deprecations/index.html)
- Compare [https://pillow.readthedocs.io/en/stable/deprecations.html](https://pillow.readthedocs.io/en/stable/deprecations.html) and [https://docs.pytest.org/en/stable/deprecations.html](https://docs.pytest.org/en/stable/deprecations.html)
```

File: docs/monthly-meeting/2024-08.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+  - duplication: ensure that the deprecation notes only exist in a single place without duplication (links to it and includes in multiple places are fine)
+  - discoverability: making sure that people can easily find what is being deprecated and removed, and how to fix it
+  - future-proofing: make sure these info are not completely removed from the latest docs (a link to older versions is fine)
+  - layout: list vs table (see https://github.com/python/cpython/issues/122652), next step is to create a table as a proof of concept to see what works best
+- *Some additional links:*
+  - python-dev thread from a decade ago: https://mail.python.org/pipermail/python-dev/2011-October/114199.html (some of these things are fixed, some are no longer relevant, some still need work)
+  - Enforcing the use of deprecated-removed: https://github.com/python/cpython/issues/92564
+  - A related core-workflow issue: https://github.com/python/core-workflow/issues/468
+  - Moving deprecations into include files: https://github.com/python/cpython/issues/122085
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:73]:
```suggestion
  - layout: list vs table (see [python/cpython#122652](https://github.com/python/cpython/issues/122652)), next step is to create a table as a proof of concept to see what works best
- *Some additional links:*
  - python-dev thread from a decade ago: [https://mail.python.org/pipermail/python-dev/2011-October/114199.html](https://mail.python.org/pipermail/python-dev/2011-October/114199.html)  (some of these things are fixed, some are no longer relevant, some still need work)
  - Enforcing the use of deprecated-removed: [python/cpython#92564](https://github.com/python/cpython/issues/92564)
  - A related core-workflow issue: [python/core-workflow#468](https://github.com/python/core-workflow/issues/468)
  - Moving deprecations into include files: [python/cpython#122085](https://github.com/python/cpython/issues/122085)
```

File: docs/monthly-meeting/2024-08.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+[#118891 Slow actions for doc builds](https://github.com/python/cpython/issues/118891)
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, etc.*
+
+### 'Internal' items
+
+*For and about the Community or Working Group*
+
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:92]:
```suggestion
```

**Review by hugovk — COMMENTED** [review]
Thank you for the PR!

**Review by hugovk — APPROVED** [review]
Thank you!

File: docs/monthly-meeting/2024-08.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+- Usman
+- Carol / `@willingc`
+- Daniele / `@EvilDMP`
+
+## Reports and celebrations
+
+- [Manuel] Moving the docs to Read the Docs
+  - Several reasons: be more open (let everyone see if the builds failed, how the builds work); faster builds;
+  - A test is already working for pull requests. Only PRs that touch documentation are built.
+  - Test: https://cpython-previews.readthedocs.io/en/main/
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:36]:
```suggestion
  - Test: <https://cpython-previews.readthedocs.io/en/main/>
```

File: docs/monthly-meeting/2024-08.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+  - 3.13 or 3.14 is the version we'll migrate first, to get a feel for the workflow. All other versions will be served by python.org for now. Then we'll migrate one version at a time.
+  - [Trey] Will this enhance the search? [Manuel] There have been many improvements in RTD search lately. Traditionally RTD overwrites Sphinx search entirely with server-side (Elastic Search) search. Now there's a way to choose RTD or Sphinx search from the docs theme. There's also a person who said they're working on the Sphinx sphinx search engine.
+  - [Trey] So there won't be a change on day one.
+  - [Carol] For the PR previews, is there an option to link to the most changed file in the PR? [Manuel] We're working on it, don't know the details. The idea is to perform a diff and determine what changed, and link to it directly.
+    - Issue: [readthedocs/readthedocs.org#11319](https://github.com/readthedocs/readthedocs.org/issues/11319)
+    - Design doc:[readthedocs/readthedocs.org#11507](https://github.com/readthedocs/readthedocs.org/pull/11507)
+  - [Carol] That's way further ahead than I thought.Thank you!
```
Comment (hugovk) [inline docs/monthly-meeting/2024-08.md:46]:
```suggestion
    - Design doc: [readthedocs/readthedocs.org#11507](https://github.com/readthedocs/readthedocs.org/pull/11507)
  - [Carol] That's way further ahead than I thought. Thank you!
```

**Review by hugovk — COMMENTED** [review]
Just spotted a couple of little things

**Comment by trallard** [comment]
Thanks for being so kind and patient. Promise I will do a deep check next time!

**Comment by hugovk** [comment]
All good! I learned we can do `<https://example.com>` to autolink plain URLs with Myst, after we'd been doing `[https://example.com](https://example.com)` for such a long time :)

**Pull request merged**